ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp

```diff
@@ -67,6 +67,20 @@ void bind_unary_composite_optional_floats_with_default(
         module,
         operation,
         doc,
```

Comment:
Can you add supported parameters as well?

Reply:
<img width="961" alt="Screenshot 2024-11-05 at 9 08 25 AM" src="https://github.com/user-attachments/assets/5f538ca5-2ff3-4036-8e04-9f1a852c3c61">

Reply:
<img width="769" alt="Screenshot 2024-11-05 at 9 09 07 AM" src="https://github.com/user-attachments/assets/344c2897-f3c6-49e0-ba76-d120b3dbbe17">